torch/ao/quantization/experimental/fake_quantize_function.py

```diff
@@ -0,0 +1,28 @@
+import torch
+from torch import Tensor
+from torch.ao.quantization.experimental.quantizer import quantize_APoT, dequantize_APoT
+
+class fake_quantize_function(torch.autograd.Function):
+    @staticmethod
+    def forward(ctx,  # type: ignore[override]
+                x: Tensor,
+                alpha: Tensor,
+                gamma: Tensor,
+                quantization_levels: Tensor,
+                level_indices: Tensor) -> Tensor:
+        quantized_result = quantize_APoT(x, alpha, gamma, quantization_levels, level_indices)
+
+        # calculate mask tensor
+        mask = torch.clone(x)
```

Comment:
why mask needs to be initialized with a clone of x?

Reply:
i used a clone because `apply` modifies tensors in-place. i can just apply on x here though, since the new x would be out of scope after the function ends.

Reply:
mask is 0s or 1s. while it's going to get overwritten later, it's not an intuitive initialization. you can just initialize it to 0 or leave it uninitialized

Reply:
yep sounds good!